src/widget/top_menu.c

```diff
@@ -428,24 +428,28 @@ static void menu_file_exit_game(int param)
 static void menu_options_display(int param)
 {
     clear_state();
+    window_city_show();
```

Comment:
This makes the "Pause" bar show behind the dialog if you have the game on pause. That does not quite look good...